style.css

```diff
@@ -2,110 +2,566 @@ html,
 body {
   font-family: "Poppins", sans-serif;
   font-style: normal;
-  background: #fff;
+  background: #e5e5e5;
+  margin: 0 0;
+  padding: 0 0;
 }
 
-header {
-  position: absolute;
-  height: 718px;
-  width: 375px;
-  left: 0;
-  top: 0;
-  right: 0;
-  padding-left: 24px;
-  padding-right: 24px;
-  padding-top: 20px;
+nav {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  margin-bottom: 0;
+  background-color: white;
+}
+
+nav .logo h3 {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+  padding-left: 155px;
+}
+
+nav ul {
+  display: flex;
+  gap: 32px;
+  list-style: none;
+  padding-right: 155px;
 }
 
-section {
-  padding-left: 24px;
-  padding-right: 24px;
-  padding-top: 120px;
-  background-image: url("images/headershapes-mobile.svg");
+nav ul li a {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  color: #344563;
+  text-decoration-line: none;
+}
+
+.Introduction {
+  width: 100%;
+  height: auto;
+  padding-top: 218px;
+  background: url(images/background.png);
   background-repeat: no-repeat;
-  background-attachment: fixed;
-  background-position: center;
+  background-size: cover;
+  border-bottom-left-radius: 89px;
+  padding-bottom: 12rem;
+}
+
+.Introduction h1 {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 48px;
+  align-items: center;
+  letter-spacing: 0.37px;
+  color: #091e42;
+  padding-left: 226px;
+  padding-bottom: 0;
+  margin-bottom: 0;
 }
 
-.wrapper {
-  padding-top: 20px;
+.Introduction p {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  color: #344563;
+  padding-left: 226px;
+  margin-bottom: 0;
 }
 
-.nav-bar {
+.Introduction .social-links-title {
+  width: 160px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  color: #7f8cff;
+  padding-bottom: 0.5rem;
+}
+
+.Introduction .social-links {
   display: flex;
+  align-items: center;
+  width: 200px;
+  padding-left: 14rem;
   justify-content: space-between;
-  margin-bottom: 80px;
 }
 
-.nav-bar a {
-  width: 76px;
-  height: 20px;
-  font-weight: 700;
-  font-size: 18px;
-  line-height: 20px;
+.container.work-area {
+  width: 100%;
+  background: #e5e5e5;
+  display: grid;
+  grid-template-rows: 1;
+  padding-left: 0;
+  padding-top: 0;
+  margin-top: 90px;
+  margin-bottom: 120px;
+}
+
+.flex-row {
+  margin-left: 0%;
+  padding-left: 16px;
+  padding-top: 10px;
   display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: flex-start;
+  row-gap: 99px;
+  column-gap: 10px;
+  border: 1px solid #dfe1e6;
+  border-radius: 24px;
+}
+
+.flex-col {
+  display: flex;
+  background: #fff;
+  border-radius: 24px;
+  height: auto;
+  margin-left: auto;
+  margin-right: auto;
+  padding-bottom: 1rem;
+  padding-right: 1.5rem;
+}
```

Comment:
- [ ] You have done great to ensure that the contents of the card are displayed flex. However, this style should only be applied to desktop version of your portifolio. In addition, your cards are causing horizontal scroll of the portifolio. You should also make sure that the left and right covers around 47% width of your card.
- [ ] Remember, the experience of scrolling horizontally can makes the screen judder and create a headache inducing feeling 😞 
- [ ] Please ensure that the design of the card matches exactly the design of the Figma template design in both mobile and desktop versions of your portifolio ☺️.

Your design 😎 desktop
![image](https://github.com/ethred/Portfolio/assets/99394374/226d2d72-6bda-482e-93f0-7fcb15d61168)

Mobile
![image](https://github.com/ethred/Portfolio/assets/99394374/8ee9de39-f057-406b-9491-07b4376a76a0)

Figma design 😎 
![image](https://github.com/ethred/Portfolio/assets/99394374/c7c91c7b-6836-49a0-bda5-106bd8949ec5)